configure.py

```diff
@@ -1523,7 +1523,8 @@ def main():
 
   set_grpc_build_flags()
   set_cc_opt_flags(environ_cp)
-  set_build_strip_flag()
+  if "BAZEL_NO_STRIP" not in environ_cp:
```

Comment:
Can we do environ_cp.get('BAZEL_NO_STRIP') == '1' instead just to be safe?

Reply:
@yifeif That would imply BAZEL_NO_STRIP=* would keep strip in, which would be counter-intuitive  I guess. How about environ_cp.get('BAZEL_STRIP') == '0'?

Reply:
Ah but we want the default of stripping bazel flags to stay the same without setting an env var everywhere. What you have sounds good then.